frontend/src/metabase/parameters/components/FormattedParameterValue/FormattedParameterValue.tsx

```diff
@@ -33,25 +39,42 @@ function FormattedParameterValue({
 
   const label = getLabel(displayValue);
 
-  if (
-    isFieldFilterUiParameter(parameter) &&
-    hasFields(parameter) &&
-    !isDateParameter(parameter)
-  ) {
+  const renderContent = () => {
+    if (
+      isFieldFilterUiParameter(parameter) &&
+      hasFields(parameter) &&
+      !isDateParameter(parameter)
+    ) {
+      return (
+        <ParameterFieldWidgetValue
+          fields={getFields(parameter)}
+          value={value}
+          displayValue={label}
+        />
+      );
+    }
+
+    if (label) {
+      return <span>{formatParameterValue(label, parameter)}</span>;
+    }
+
+    return <span>{formatParameterValue(value, parameter)}</span>;
+  };
+
+  if (isStringParameter(parameter)) {
+    const hasLongValue = typeof first === "string" && first.length > 80;
     return (
-      <ParameterFieldWidgetValue
-        fields={getFields(parameter)}
-        value={value}
-        displayValue={label}
-      />
+      <Ellipsified
+        showTooltip={!isPopoverOpen}
+        multiline
+        tooltipMaxWidth={hasLongValue ? 450 : undefined}
+      >
+        {renderContent()}
+      </Ellipsified>
     );
   }
```

Comment:
Labels shorter that 80 characters look odd when `tooltipMaxWidth` is set:


![CleanShot 2024-07-26 at 15 32 02@2x](https://github.com/user-attachments/assets/24074316-6423-4e64-a938-63ee6268732a)